data/mods/Xedra_Evolved/ui/vamp_blood_levels.json

```diff
@@ -0,0 +1,88 @@
+[
+  {
+    "id": "vampire_blood_levels",
+    "type": "widget",
+    "label": "Blood",
+    "style": "text",
```

Comment:
```suggestion
    "style": "text",
    "flags": [ "W_DISABLED_WHEN_EMPTY" ],
```

Reply:
How does that work with negative numbers? Will it be disabled when negative?

Reply:
It means the widget is disabled only if there's nothing to display, i.e. when none of the clauses are matched. So if the player has no `vampire_virus` effect, none of the clauses here will match and the widget will be disabled